src/jdk.jdwp.agent/share/native/libjdwp/threadControl.c

```diff
@@ -2179,6 +2180,59 @@ threadControl_onEventHandlerEntry(jbyte sessionID, EventInfo *evinfo, jobject cu
 static void
 doPendingTasks(JNIEnv *env, ThreadNode *node)
 {
+    /* Deferred breakpoint handling for Thread.resume() */
+    if (node->handlingAppResume) {
+        jthread resumer = node->thread;
+        jthread resumee = getResumee(resumer);
+
+        /*
+         * trackAppResume() (indirectly) aquires handlerLock. For proper lock
```

Comment:
I'd suggest to get rid of '()' brackets in this comment line, so it will be:
   * trackAppResume indirectly aquires handlerLock. For proper lock

Reply:
Sure. Done.